docs/sdk/serverless-signing/uploading.md

```diff
@@ -0,0 +1,230 @@
+# Uploading Your Lit Action
+
+Lit Actions are powerful JavaScript programs that run on a Lit network. This guide covers the two processes for providing your Lit Action code to the Lit network. For more info on what a Lit Action is, go [here](./overview.md.). For a guide on getting started with Lit Actions, go [here](./quick-start.md).
```

Comment:
> This guide covers the two processes for providing your Lit Action code to the Lit network

consider "this guide covers the two approaches you can use to deploy your Lit Action with Lit"